nixos/tests/drbd-driver.nix

```diff
@@ -0,0 +1,19 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: {
+  name = "drbd-driver";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ birkb ];
+  };
+
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      boot.kernelModules = [ "drbd" ];
+      boot.extraModulePackages = with config.boot.kernelPackages; [ drbd ];
+      boot.kernelPackages = pkgs.linuxPackages;
```

Comment:
```suggestion
      boot = {
        kernelModules = [ "drbd" ];
        extraModulePackages = with config.boot.kernelPackages; [ drbd ];
        kernelPackages = pkgs.linuxPackages;
      };
```

Reply:
Hi Astro, thanks for the hint. I have tweaked both areas.